config.toml

```
@@ -0,0 +1,329 @@
+koji_target = 'f39-pythonxx'
```

Let's avoid all top-level configuration?

```suggestion
[koji]
target = 'f39-pythonxx'
```